src/Microsoft.DotNet.Build.Tasks/PackageFiles/packageresolve.targets

```diff
@@ -35,19 +35,21 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <ResolveAssemblyReferencesDependsOn>
+    <ResolveAssemblyReferencesDependsOn Condition="'$(_BuildAgainstPackages)' != 'true'">
       $(ResolveAssemblyReferencesDependsOn);
       ResolveNuGetPackages;
       ValidatePackageVersions;
     </ResolveAssemblyReferencesDependsOn>
+ </PropertyGroup>
 
+ <Import Condition="'$(_BuildAgainstPackages)' == 'true'" Project="buildagainstpackages.targets" />
```

Comment:
If _BuildAgainstPackages is intented to be passed as a command line parameter I would prefer we remove the "_".